docs/examples/basic_meter/basic_metrics.py

```diff
@@ -42,6 +43,11 @@ def usage(argv):
         print("bad mode specified.")
         usage(sys.argv)
         sys.exit(1)
+    stateful = batcher_mode == "stateful"
+
+print(
+    "Starting example, values will be printed to the console each 5 seconds."
```

Comment:
I (non native english speaker) think it should be `every 5 seconds`
```suggestion
    "Starting example, values will be printed to the console every 5 seconds."
```